packages/fresh_graphql/example/main.dart

```diff
@@ -1,40 +0,0 @@
-import 'dart:io';
```

Comment:
why did we delete the example?

Reply:
I accidentally deleted this file. Recovered file.